app/src/androidTest/java/net/skyscanner/backpack/compose/imagegallery/BpkImageGalleryTest.kt

```diff
@@ -28,4 +33,29 @@ class BpkImageGalleryTest : BpkSnapshotTest() {
     fun carousel() {
         snap { ImageGalleryCarouselStory() }
     }
+
+    @Test
+    fun slideshow() {
+        recordModal { ImageGallerySlideshowStory() }
+    }
+
+    @Test
+    fun slideshow_landscape() {
+        val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
```

Comment:
we have a different layout for landscape - make sure we switch to landscape first